src/py42/services/auditlogs.py

```diff
@@ -29,14 +33,21 @@ def get_page(self, page_num, page_size=None, **kwargs):
             page=page_num - 1,
             pageSize=page_size,
             dateRange=date_range,
-            eventTypes=event_types,
-            actorIds=user_ids,
-            actorNames=usernames,
-            actorIpAddresses=user_ip_addresses,
-            affectedUserIds=affected_user_ids,
-            affectedUserNames=affected_usernames,
-            **kwargs
+            eventTypes=to_list(kwargs.get(u"event_types")),
```

Comment:
Why not just have the expected params be listed out in the definition? `kwargs` is just for unexpected params, like ones we don't support yet.

Reply:
changed.